AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -263,12 +262,10 @@ struct ChooseSessionTypeView: View {
             }
             .padding([.bottom, .vertical])
             .padding(.horizontal, 30)
-            .background(
-                Color.aircastingBackground.opacity(0.25)
-                    .ignoresSafeArea()
-            )
+            .background(Color.aircastingSecondaryBackground.ignoresSafeArea())
             .alert(item: $viewModel.alert, content: { $0.makeAlert() })
         }
+        .background(Color.aircastingBackgroundWhite.ignoresSafeArea())
```

Comment:
Do you need this one?

Reply:
Yes, I need this one for the background on the top as on the attached image.

![Simulator Screen Shot - iPhone 11 - 2022-07-26 at 09 36 31](https://user-images.githubusercontent.com/82096200/180950358-5ee3cb33-34b1-48f3-8d1d-a87f4cbed10c.png)